examples/create-maps-with-query/create_maps_with_query.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+type Student record {|
+    readonly int id;
+    string firstName;
+    string lastName;
+    int score;
+|};
+
+public function main() returns error? {
+    table<Student> key(id) students = table [
+            {id: 1, firstName: "John", lastName: "Smith", score: 100},
+            {id: 2, firstName: "Jane", lastName: "Smith", score: 150},
+            {id: 4, firstName: "Fred", lastName: "Bloggs", score: 200},
+            {id: 7, firstName: "Bobby", lastName: "Clark", score: 300},
+            {id: 9, firstName: "Cassie", lastName: "Smith", score: 250}
+        ];
+
+    // The type of the value in the select clause must belong to the tuple type
```

Comment:
```suggestion
    // The type of the value in the `select` clause must belong to the tuple type
```

Reply:
In this context, `select` is not treated as a keyword right? It is the name of the clause. So do we need to highlight it as code in this context?

Reply:
Resolved offline to update as suggested